components/proxy/Dockerfile

```diff
@@ -25,6 +27,8 @@ RUN git clone -b http2 https://github.com/aledbf/caddy caddy-fork
 RUN xcaddy build \
   --output /caddy \
   --with github.com/caddyserver/caddy/v2=$PWD/caddy-fork \
+  --with github.com/leodido/caddy-conditional-logging \
+  --with github.com/leodido/caddy-jsonselect-encoder \
```

Comment:
Please pull the plugins into this repo. That would also solve the leeway rebuilding issue.